subsystem/swerveShooter.py

```diff
@@ -0,0 +1,18 @@
+import commands2
+import rev
+import wpilib
+
+class SwerveShooter(commands2.SubsystemBase):
+    def __init__(self) -> None:
+        self.intakeMotor = rev.CANSparkMax(23, rev.CANSparkLowLevel.MotorType.kBrushless)
```

Comment:
should rename to shooterIntakeMotor for clarity

Reply:
Already has context. Renamed the main class to ShortShooter since this is only for this year robot. Could be more specific but the contex for the subsystem I think is strong